operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -850,6 +850,11 @@ ingester:
     # Useful if the volume's root directory is not empty
     subPath: ''
 
+    # -- Enable StatefulSetAutoDeletePVC feature
```

Comment:
Can you add a link to the kubernetes feature? I think the same link that you have in the PR description will suffice.

Reply:
Let me know if you prefer add this other [link](https://kubernetes.io/blog/2023/05/04/kubernetes-1-27-statefulset-pvc-auto-deletion-beta/) instead. It has some examples.